src/main/java/at/hannibal2/skyhanni/features/misc/chocolatefactory/HoppityEggsManager.kt

```diff
@@ -0,0 +1,112 @@
+package at.hannibal2.skyhanni.features.misc.chocolatefactory
+
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.events.LorenzWorldChangeEvent
+import at.hannibal2.skyhanni.features.fame.ReminderUtils
+import at.hannibal2.skyhanni.test.command.ErrorManager
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.DelayedRun
+import at.hannibal2.skyhanni.utils.LocationUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.renderStrings
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+object HoppityEggsManager {
+
+    private val config get() = ChocolateFactoryApi.config.hoppityEggs
+
+    private val eggFoundPattern by ChocolateFactoryApi.patternGroup.pattern(
+        "egg.found",
+        "§d§lHOPPITY'S HUNT §r§dYou found a §r§.Chocolate (?<meal>\\w+) Egg.*"
+    )
+    private val noEggsLeftPattern by ChocolateFactoryApi.patternGroup.pattern(
+        "egg.noneleft",
+        "§cThere are no hidden Chocolate Rabbit Eggs nearby! Try again later!"
+    )
+    private val eggSpawnedPattern by ChocolateFactoryApi.patternGroup.pattern(
+        "egg.spawned",
+        "§d§lHOPPITY'S HUNT §r§dA §r§.Chocolate (?<meal>\\w+) Egg §r§dhas appeared!"
+    )
+    private val eggAlreadyCollectedPattern by ChocolateFactoryApi.patternGroup.pattern(
+        "egg.alreadycollected",
+        "§cYou have already collected this Chocolate (?<meal>\\w+) Egg§r§c! Try again when it respawns!"
+    )
+
+    private var lastMeal: HoppityEggType? = null
+
+    @SubscribeEvent
+    fun onWorldChange(event: LorenzWorldChangeEvent) {
+        lastMeal = null
+    }
+
+    @SubscribeEvent
+    fun onChat(event: LorenzChatEvent) {
+        if (!LorenzUtils.inSkyBlock) return
```

Comment:
misses an enable

Reply:
there is no enable here as i need this data for if they turn it back on later